pyramid/viewderivers.py

```diff
@@ -6,7 +6,7 @@
     )
 
 from pyramid.security import NO_PERMISSION_REQUIRED
-from pyramid.session import check_csrf_token
+from pyramid.session import check_csrf_origin, check_csrf_token
```

Comment:
Same here, imports across multiple lines, alphabetical order. Most of the Pyramid source code has that for multiple imports.